### PR TITLE
Add walkthrough and screenshots to creating workflows

### PR DESCRIPTION
**Pull request by tonyhb**

### CODE REVIEW

File: docs/workflows/creating_workflows.md [hunk 1]
```diff
@@ -14,11 +14,36 @@ You can create new workflows in one of two ways:
...
+3. Choose the type of workflow you'd like to make.  **Event** workflows trigger on events, and **scheduled** worfklows run on a schedule.
+
+    a. For event workflows, type in the name of the event that should trigger this workflow.  The autocomplete utility will show you all received events.  If you want another event to trigger the workflow type the name in and ignore the autocomplete.
+
+    b. For scheduled workflows, add a cron expression for the schedule.  [This website](https://crontab.guru/) can help you with your expressions.
+
+4. From there, you'll see our workflow editor.  Click the "add" button to add new actions that run within your workflow.
+
+<div>
+	<img src="/create-workflow.gif" alt="Creating a workflow from the UI" />
```
Comment (rashiq) [inline docs/workflows/creating_workflows.md:38]: is it just me or is the create-workflow.gif quite low resolution? ;_:

File: docs/workflows/creating_workflows.md [hunk 2]
```diff
@@ -14,11 +14,36 @@ You can create new workflows in one of two ways:
...
+2. Hit the "New workflow" button in the top right corner
+
+<div>
+	<img src="/create-workflow.png" alt="Creating a workflow from the UI" />
+	<small>Hit "workflows", then New from the top right to create a new workflow.</small>
+</div>
+
+3. Choose the type of workflow you'd like to make.  **Event** workflows trigger on events, and **scheduled** worfklows run on a schedule.
+
+    a. For event workflows, type in the name of the event that should trigger this workflow.  The autocomplete utility will show you all received events.  If you want another event to trigger the workflow type the name in and ignore the autocomplete.
```
Comment (rashiq) [inline docs/workflows/creating_workflows.md:31]: nit: show you all received events **[so far]**?

File: docs/workflows/creating_workflows.md [hunk 3]
```diff
@@ -14,11 +14,36 @@ You can create new workflows in one of two ways:
...
+
+4. From there, you'll see our workflow editor.  Click the "add" button to add new actions that run within your workflow.
+
+<div>
+	<img src="/create-workflow.gif" alt="Creating a workflow from the UI" />
+	<small>Add and customize actions within your workflow.</small>
+</div>
+
+5. Create your workflow by adding and configuring your actions, then hit "Save draft" to save the draft or "Publish workflow" to push the workflow live.
+6. You can also test the workflow by hitting "Run" below the save action.  Paste tha event data that will trigger the workflow on the left, and hit "Run" from the debug bar to preview your workflow.
```
Comment (rashiq) [inline docs/workflows/creating_workflows.md]:
```suggestion
6. You can also test the workflow by hitting "Run" below the save action.  Paste the event data that will trigger the workflow on the left, and hit "Run" from the debug bar to preview your workflow.
```

**Review by rashiq — APPROVED** [review]

File: docs/workflows/creating_workflows.md (hunk 3 above)
Comment (rashiq) [reply docs/workflows/creating_workflows.md]: Sorry meant to suggest change but accidentally actually committed it to the branch lol

File: docs/workflows/creating_workflows.md (hunk 1 above)
Comment (tonyhb) [reply docs/workflows/creating_workflows.md:38]: i shall fix on ffmpeg!

**Pull request merged**